js/vendor.webpack.config.js

```diff
@@ -0,0 +1,43 @@
+var webpack = require('webpack');
+
+var ENV = process.env.NODE_ENV || 'development';
+var isProd = ENV === 'production';
+
+module.exports = {
+  entry: {
+    vendor: [
+      'react', 'react-dom', 'react-redux', 'react-router',
+      'redux', 'redux-thunk', 'react-router-redux',
+      'lodash', 'material-ui', 'blockies'
+    ]
```

Comment:
Probably think we should remove 'babel-polyfill' from commons, add it here as well.